common/assets/scss/application.scss

```diff
@@ -22,20 +22,23 @@ $govuk-font-family: "Inter", system-ui, sans-serif;
 @import "govuk-frontend/govuk/components/input/input.scss";
 @import "govuk-frontend/govuk/components/inset-text/inset-text.scss";
 @import "govuk-frontend/govuk/components/panel/panel.scss";
-@import "govuk-frontend/govuk/components/phase-banner/phase-banner.scss";
 @import "govuk-frontend/govuk/components/radios/radios.scss";
 @import "govuk-frontend/govuk/components/select/select.scss";
 @import "govuk-frontend/govuk/components/skip-link/skip-link.scss";
 @import "govuk-frontend/govuk/components/summary-list/summary-list.scss";
 @import "govuk-frontend/govuk/components/textarea/textarea.scss";
 @import "govuk-frontend/govuk/components/warning-text/warning-text.scss";
 
+
```

Comment:
Any reason for this?

Reply:
You said we wanted to drop the phase banner because otherwise it would have been too busy with the new navigation...

Reply:
Sorry, I just meant the addition of another new line.